src/Dapr.Jobs/Extensions/ByteArrayDeserializationExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// ------------------------------------------------------------------------
+// Copyright 2024 The Dapr Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//     http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ------------------------------------------------------------------------
+
+using System.Text;
+using System.Text.Json;
+
+namespace Dapr.Jobs.Extensions;
+
+/// <summary>
+/// Provides utility extensions for deserializing an array of UTF-8 encoded bytes.
+/// </summary>
+public static class ByteArrayDeserializationExtensions
```

Comment:
`internal`?

Reply:
Did you one better - removed altogether since its only usage anymore appeared to be a unit test (removed them as well).